tests/integ/sagemaker/jumpstart/model/test_jumpstart_model.py

```diff
@@ -262,3 +260,25 @@ def test_jumpstart_model_register(setup):
     response = predictor.predict("hello world!")
 
     assert response is not None
+
+
+@pytest.mark.skipif(
+    True,
+    reason="Only enable if test account is subscribed to the proprietary model",
+)
+def test_proprietary_jumpstart_model(setup):
+
+    model_id = "ai21-jurassic-2-light"
+
+    model = JumpStartModel(
+        model_id=model_id,
```

Comment:
pin the version here please, otherwise it might fail if ai21 releases a new version.

Reply:
I am afraid I must insist

Reply:
Sorry was removed during testing, yes added version